app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -3,8 +3,8 @@ import React from "react";
 import dayjs from "dayjs";
 import * as yup from "yup";
 
-import Avatar from "./Contact/Avatar";
-import DetailText from "./Contact/DetailText";
+import { Avatar } from "./Contact";
+import { DetailText } from "./Contact";
```

Comment:
The whole point of adding an index.js was to import in a single line:
```suggestion
import { Avatar, DetailText } from "./Contact";
```